services/app-web/src/pages/StateSubmission/Documents/Documents.tsx

```diff
@@ -44,20 +44,48 @@ export const Documents = ({
     const hasValidFiles = fileItems.every(
         (item) => item.status === 'UPLOAD_COMPLETE'
     )
+    const hasMissingCategories =
+        /* fileItems must have some document category.  a contract-only submission
+       must have "CONTRACT_RELATED" as the document category. */
+        fileItems.length > 0 &&
+        (fileItems.some((docs) => docs.documentCategories.length === 0) ||
+            (isContractOnly &&
+                fileItems.some(
+                    (docs) =>
+                        !docs.documentCategories.includes('CONTRACT_RELATED')
+                )))
     const hasLoadingFiles =
         fileItems.some((item) => item.status === 'PENDING') ||
         fileItems.some((item) => item.status === 'SCANNING')
     const showFileUploadError = shouldValidate && !hasValidFiles
 
+    const documentsErrorMessages = () => {
+        const errorsObject: { [k: string]: string } = {}
+        fileItems.forEach((item) => {
+            const key = item.id
+            if (item.status === 'DUPLICATE_NAME_ERROR') {
+                errorsObject[key] = 'You must remove duplicate files'
+            } else if (item.status === 'SCANNING_ERROR') {
+                errorsObject[key] =
+                    'You must remove files that failed the security scan'
+            } else if (item.status === 'UPLOAD_ERROR') {
+                errorsObject[key] =
+                    'You must remove or retry files that failed to upload'
+            } else if (item.documentCategories.length === 0) {
+                errorsObject[key] = 'You must select a document category'
+            }
+        })
+        return errorsObject
```

Comment:
I think this might be the place that is causing the differences seen here https://github.com/CMSgov/managed-care-review/pull/799#issuecomment-1039182993